packages/suite/src/components/wallet/PrivacyAccount/AnonymityLevelSlider.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useCallback, useState, ChangeEventHandler } from 'react';
+import styled from 'styled-components';
+import { Range, P, variables } from '@trezor/components';
+import { useSelector, useActions } from '@suite-hooks';
+import { selectSelectedAccount } from '@wallet-reducers/selectedAccountReducer';
+import { selectCurrentTargetAnonymity } from '@wallet-reducers/coinjoinReducer';
+import * as coinjoinActions from '@wallet-actions/coinjoinAccountActions';
+
+const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+const Slider = styled(Range)`
+    padding-bottom: 50px;
+    padding-top: 60px;
+    margin-top: 0;
+
+    cursor: pointer;
+    background: none;
+`;
+
+const LabelsWrapper = styled.div`
+    display: flex;
+    justify-content: space-between;
+    pointer-events: none;
+
+    margin-top: -46px;
+    margin-bottom: -16px;
+    margin-left: 10px;
+`;
+
+const Label = styled(P)<{ $offset?: number }>`
+    position: relative;
+    left: ${({ $offset }) => $offset ?? 0}px;
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    opacity: 0.5;
+`;
+
+interface AnonymityLevelSliderProps {
+    className?: string;
+}
+
+const trackStyle = {
+    background:
+        'linear-gradient(270deg, #2A9649 0%, #95CDA5 45%, #C8B883 55%, #C8B882 60%, #BF6767 100%);',
+};
+
+const minPosition = 0;
+const maxPosition = 100;
+
+const minValue = Math.log(1);
+const maxValue = Math.log(100);
+
+const scale = (maxValue - minValue) / (maxPosition - minPosition);
+
+const getValue = (position: number) =>
+    Math.round(Math.exp((position - minPosition) * scale + minValue));
+const getPosition = (value: number) => minPosition + (Math.log(value) - minValue) / scale;
+
+export const AnonymityLevelSlider = ({ className }: AnonymityLevelSliderProps) => {
+    const currentAccount = useSelector(selectSelectedAccount);
+    const targetAnonymity = useSelector(selectCurrentTargetAnonymity);
+
+    const { coinjoinAccountUpdateAnonymity } = useActions({
+        coinjoinAccountUpdateAnonymity: coinjoinActions.coinjoinAccountUpdateAnonymity,
+    });
+
+    const [sliderPosition, setSliderPosition] = useState(getPosition(targetAnonymity || 1));
+
+    const handleSliderChange: ChangeEventHandler<HTMLInputElement> = useCallback(
```

Comment:
Seems like right place for `useTransition` or `startTransition`. Let's start to use React 18 features since we have them!

Reply:
We also can use this as an example right away. What do you say, Matej, wanna be the R18 trailblazer? 🧠

Reply:
Is this the right way of using it? https://github.com/trezor/trezor-suite/pull/6569/commits/59e2b7125a9d7d23df80dc467602d0d65eef2c78 

Not sure if I should wrap only `setSliderPosition` by startTransition or the whole handler as I did. @Nodonisko what do you think?

Reply:
I think only the `coinjoinAccountUpdateAnonymity()` call should be wrapped, since we want the `sliderPosition` to be updated immediately to reflect the position in the UI but not necessarily the state value.

Reply:
It seems like there is no effect, since redux dispatch is not handled by the transition API :C Tried it, the event it still dispatched way to many times. But for React state changes – definitely the way to go instead of just debouncing! Correct me if I missed something @Nodonisko 

Visually it would not be so smooth but updating the Redux state that often for observing numbers changing quickly seems like an overkill, lags when the browser is throttled.